astropy/visualization/tests/test_units.py

```diff
@@ -40,6 +40,28 @@ def test_units():
         assert plt.gca().yaxis.get_units() == u.kg
 
 
+@pytest.mark.skipif(not HAS_PLT, reason="requires matplotlib.pyplot")
+def test_units_decorator():
+    @quantity_support()
+    def run_test():
+        plt.figure()
+
+        buff = io.BytesIO()
+
+        plt.plot([1, 2, 3] * u.m, [3, 4, 5] * u.kg, label="label")
+        plt.plot([105, 210, 315] * u.cm, [3050, 3025, 3010] * u.g)
+        plt.legend()
+        # Also test fill_between, which requires actual conversion to ndarray
+        # with numpy >=1.10 (#4654).
```

Comment:
numpy 1.10 is long gone by now, I suggest dropping this historical comment in new code
```suggestion
```

Reply:
Done, removed from the new test and the test that I copied it from.